src/lib/schematics/ng-update/upgrade-rules/package-imports-v8/updateAngularMaterialImportsRule.ts

```diff
@@ -0,0 +1,172 @@
+/**
+ * This schematic corrects to importing symbols from Angular Material through
+ * subpackages (e.g. @angular/material/button) rather than the top level
+ * package (e.g. @angular/material).
+ */
+import * as Lint from 'tslint';
+import * as tsutils from 'tsutils';
```

Comment:
I think we need to avoid using `tsutils` as it is not guaranteed to be part of the CLI project right?